inlong-dashboard/src/pages/Clusters/config/Pulsar.tsx

```diff
@@ -29,9 +30,16 @@ export const Pulsar: ClsConfigItemType[] = [
   {
     type: 'input',
     label: 'ServiceUrl',
-    name: 'serviceUrl',
+    name: 'url',
```

Comment:
url
->
adminUrl

Reply:
The `url` was the backend requirement, it's a general name.